### PR TITLE
Bump axios due to security vulnerability

### PR DESCRIPTION
**Pull request by rcarmo**
https://github.com/axios/axios/pull/3410

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Checklist

- [x] I have read the [contribution guidelines](https://github.com/node-red/node-red/blob/master/CONTRIBUTING.md)
- [ ] For non-bugfix PRs, I have discussed this change on the forum/slack team.
- [ ] I have run `grunt` to verify the unit tests pass
- [ ] I have added suitable unit tests to cover the new/changed functionality

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,8 +28,8 @@
         "test": "./node_modules/.bin/mocha "
     },
     "dependencies": {
-        "axios": "0.21.1",
-        "bcryptjs": "^2.4.3",
+        "axios": "^0.21.1",
```
Comment (dceejay) [inline package.json:31]: How does this fix the issue ? Why not ensure it is the correct level to fix the issue raised ?

File: package.json [hunk 2]
```diff
@@ -28,8 +28,8 @@
         "test": "./node_modules/.bin/mocha "
     },
     "dependencies": {
-        "axios": "0.19.2",
-        "bcryptjs": "^2.4.3",
+        "axios": "^0.21.1",
+        "bcryptjs": "^5.0.0",
```
Comment (knolleary) [inline package.json:32]: There is no `5.x` version of `bcryptjs` - the latest is `2.4.3` which is the version we had listed already.

File: package.json [hunk 3]
```diff
@@ -28,8 +28,8 @@
         "test": "./node_modules/.bin/mocha "
     },
     "dependencies": {
-        "axios": "0.21.1",
-        "bcryptjs": "^2.4.3",
+        "axios": "^0.21.1",
+        "bcryptjs": "^5.0.0",
```
Comment (dceejay) [inline package.json:32]: we still support nodejs 8 - this change drops support for 8 - so we can't go there until we do a major update. See https://discourse.nodered.org/t/bcrypt-update-from-3-0-6-to-5-0-0/32693

Comment (knolleary) [reply package.json:32]: This dependency is `bcryptjs` **not** `bcrypt`. There is *nothing* to change about `bcryptjs`.

Your point is valid about `bcrypt` - but this is *not* that module being changed here - but the change is still invalid as per my other review comment.

**Comment by knolleary** [comment]
Resolved via #13

**Pull request closed without merging**